Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 ﻿note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
```

Comment:
I guess this is why we started the discussion about making class TYPE expanded.
I'm sure that that there is code out there which relies on the fact that {ANY} = {ANY}.
Note that the ECMA meeting minutes mentions the singleton property of TYPE, although I could not find it in the standard itself.